integration-tests/src/test/scala/org/constellation/DownloadSpec.scala

```diff
@@ -0,0 +1,98 @@
+package org.constellation
+
+import cats.effect.{ContextShift, ExitCode, IO, Timer}
+import com.decodified.scalassh._
+import io.chrisdavenport.log4cats.slf4j.Slf4jLogger
+import org.constellation.schema.NodeState.Offline
+import org.constellation.schema.{ClusterNode, MetricsResult, NodeState, NodeStateInfo}
+import org.constellation.testutils.HttpUtil.{evalRequest, evalRequestForHosts}
+import org.constellation.testutils.{CustomMatchers, HttpUtil}
+import org.http4s.Uri
+import org.http4s.circe.CirceEntityDecoder._
+import org.http4s.client.dsl.io._
+import org.http4s.dsl.io._
+import org.scalatest.concurrent.PatienceConfiguration.Timeout
+import org.scalatest.concurrent.{Eventually, PatienceConfiguration}
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.time.SpanSugar._
+
+import scala.concurrent.ExecutionContext.global
+import scala.io.Source
+
+class DownloadSpec extends AnyFreeSpec with CustomMatchers with TestConfig with TerraformOutput with Eventually {
+
+  private val logger = Slf4jLogger.getLogger[IO]
+
+  override implicit val patienceConfig: PatienceConfig =
+    PatienceConfig(
+      timeout = scaled(3 minutes),
+      interval = scaled(1 second)
+    )
+
+  implicit val cs: ContextShift[IO] = IO.contextShift(global)
+  implicit val timer: Timer[IO] = IO.timer(global)
+
+  private val hostConfigProvider = HostConfigProvider
+    .fromHostConfig(
+      HostConfig(
+        login = PublicKeyLogin(config.ssh.user, config.ssh.keyfile),
+        hostKeyVerifier = HostKeyVerifiers.DontVerify
+      )
+    )
+
+  private val instanceIp: String = terraform.instanceIps.value.last
+
+  "restarting a node and joining cluster" in {
+    terraform.instanceIps.value.length should be > 3
```

Comment:
Is this assertion here to ensure that after dropping one node, the cluster won't stop making consensuses?

Reply:
Correct, that was my intention. Do you think it makes sense?

Reply:
Yeah I guess :). We know that under 3 nodes the cluster will behave differently, so I guess we should assert on the number.